Services/OutputCacheServices.cs

```diff
@@ -0,0 +1,38 @@
+﻿using RestSharp;
+using System.IO;
+
+namespace AndcultureCode.CSharp.Sitefinity.Testing.Services
+{
+    public class OutputCacheService
+    {
+        public string AuthKey { get; set; }
+        public string BaseUrl { get; set; }
+
+        public OutputCacheService(string baseUrl, string authKey)
+        {
+            AuthKey = authKey;
+            BaseUrl = baseUrl;
+        }
+
+        /// <summary>
+        /// Clears the output cache for use by the other oData services.  Use this when testing 
+        /// updates to the page services.  When POSTing, PUTing, and GETing all within a single
+        /// test in that order, Sitefinity will intermittenly returned cached results in the GET
+        /// from the original POST.  Make sure this is called before the GET is called to avoid
+        /// caching issues.
+        /// </summary>
+        /// <returns>The response of the executed request.</returns>
+        public IRestResponse Clear()
+        {
+            var client = new RestClient(Path.Combine(BaseUrl + "restapi/cache/clear"));
+
+            var request = new RestRequest(Method.POST);
+
+            request.AddHeader("SF_OUTPUTCACHE_AUTH", AuthKey); //"3y368tnfqre6PbjM4mTdRze8");
+
+            IRestResponse response = client.Execute(request);
```

Comment:
One thought, should we assert on the response here? I'm just wondering if it makes sense to fail a test that uses this method, so we can output the fact the cache failed to clear as the failure reason. Or, is it worth trying to continue anyways because sometimes it works regardless of if we manually clear the cache?

Reply:
We could but I personally think we should keep this consistent with the OData services in this repo.  They're all returning the response and allowing the consumer to handle the response.  See `ODataService.ExecuteSitefinityRequest()`.